files/zh-cn/web/api/mediaquerylist/change_event/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: 'MediaQueryList: change event'
+slug: Web/API/MediaQueryList/change_event
+page-type: web-api-event
+tags:
+  - API
+  - CSSOM View
+  - Event Handler
+  - MediaQueryList
+  - Event
+  - Reference
+  - onchange
+browser-compat: api.MediaQueryList.change_event
```

Comment:
This is used by `{{Specifications}}` and `{{Compat}}` macros, and could fall back to `en-US` documents. So we can just remove it.

```suggestion
```